src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -107,9 +107,15 @@ class VolumeBar extends Slider {
   updateARIAAttributes(event) {
     // Current value of volume bar as a percentage
     const volume = Math.round((this.player_.volume() * 100)).toString();
```

Comment:
Ah, yeah, that's much better! It occurred to me that perhaps using a conditional could make `updateARIAAttributes` even shorter and also, by extracting the statement that returns volume as a percentage into a named method, eliminate the need for an explanatory comment; please see https://github.com/videojs/video.js/pull/4099/commits/0c5f3cc5b8873c93774d885c6832152290bbfd59 and let me know what you think.